hot/patch.js

```diff
@@ -1,10 +1,12 @@
 /**
  * Copyright 2017-present, Callstack.
  * All rights reserved.
- * 
+ *
  * @flow
  */
 
+require('../src/hot/client/polyfillEnvironment');
```

Comment:
actually, why not inline all of this? it's really not that much code, but requiring through `../src` creates weird relations and will be an issue (possibly hard to catch without manually testing) when we decide to build the project instead of doing awful babel-register

Reply:
I hit these two issues when I tried to inline them. Moving the files into the same folder works though ¯\\_(ツ)_/¯

```
$ yarn test
yarn run v1.15.2
$ yarn flow && yarn lint && yarn jest
$ flow --include-warnings
Error: hot/patch.js:8
  8: import { NativeModules } from 'react-native';
                                   ^^^^^^^^^^^^^^ react-native. Required module not found

Error: hot/patch.js:61
 61:     __webpack_require__.p = `${global.DEV_SERVER_ORIGIN}/`; // eslint-disable-line no-undef
         ^^^^^^^^^^^^^^^^^^^ __webpack_require__. Could not resolve name


Found 2 errors
```